### DISCUSSION TITLE
zh/guide/drivers/uss

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#59] [Announcements]
# zh/guide/drivers/uss

AList v3的文档

https://alist.nn.ci/zh/guide/drivers/uss.html

**Comment by yangyzp** [comment]
请问，“Endpoint”这个参数在哪里找。？

**Comment by yoval** [comment]
又拍云是支持FTP管理的，用FTP也可以添加。

**Comment by wangwuxuan2011** [comment]
现在最新版本修复了又拍云部分的代码并且已经完善文档了，Endpoint可以使用加速域名。

**Comment by In2013** [comment]
新增的自定义host怎么用？

**Reply by wangwuxuan2011** [reply]
新版本目前去除了自定义host，麻烦使用最新版本并看一下文档～

**Reply by wangwuxuan2011** [reply]
Endpoint可以填入又拍云的默认加速域名或者自行绑定的域名

**Comment by qingshanking** [comment]
签名链接有效期，这个功能是否可以做一个开关，我没配置这些，文件列表后面加后缀了，导致文件都404了。

**Comment by h1k4r** [comment]
诚心发问 已开启禁止referer为空、时间戳防盗链
提示 failed get objs: failed to list objs: list error: status=401, code=40100006, message=user not exists, request-id=ad02c536db7d621a3ddc06f5a00cce9e
